### DISCUSSION TITLE
Disabling bufferline

### DISCUSSION THREAD

**Discussion by gonzus** [LunarVim/LunarVim#3499] [General]
If I want to disable breadcrumbs, adding this in my `config.lua` works: `lvim.builtin.breadcrumbs.active = false`. But I cannot disable bufferline; this line has no effect: `lvim.builtin.bufferline.active = false`.

Maybe bufferline should not be disabled because it is used internally for other stuff? If that is the case, is there a configuration setting for it that would hide the line? I find that breadcrumbs are enough for me.

**Comment by LostInTheLogs** [comment]
you need to run `:PackerSync`, and you may want to disable tabline: `vim.opt.showtabline = 0`

**Reply by gonzus** [reply]
Yes, that did it, thanks for the reminder. I also added this to my `config.lua`:
```lua
vim.opt.showtabline = 0
```
and now the tab line is completely gone.

**Reply by gonzus** [reply]
Wait, I am either blind or your comment did not mention `showtabline` when I first opened it... Anyway, thanks for the **complete** answer to my question!

**Comment by LostInTheLogs** [comment]
Yes it did not at first, i edited that part in

28 Nov 2022, 00:27 by ***@***.***:
